xbmc/utils/BitstreamConverter.cpp

```diff
@@ -425,7 +425,7 @@ bool CBitstreamConverter::Open(enum AVCodecID codec, uint8_t *in_extradata, int
       // valid hvcC data (bitstream) always starts with the value 1 (version)
       if(m_to_annexb)
       {
-       /** @todo from Amlogic
+       /**
```

Comment:
I don't know how much if any of that code can be removed.

Reply:
It looks to me like the code is used in the `xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecAndroidMediaCodec.cpp`